roles/girder-install/tasks/main.yml

```diff
@@ -1,45 +1,28 @@
 ---
-
-  - name: apt | cache | update
-    apt: update_cache=yes
-    when: do_install|bool
+  - name: girder | add | node repository
+    shell: "curl -sL https://deb.nodesource.com/setup | sudo bash -"
```

Comment:
For nodejs, there's the `nodejs` role.  I'm willing to support installing the nodesource binaries in addition to building from source, but that change would need to go in that role.